okhttp/src/main/java/okhttp3/internal/http/CallServerInterceptor.java

```diff
@@ -99,6 +99,25 @@ public CallServerInterceptor(boolean forWebSocket) {
         .responseHeadersEnd(realChain.call(), response);
 
     int code = response.code();
+    if (code == 100) {
```

Comment:
In follow up we should probably check the entire range.

Reply:
So you mean allowing this for every 1xx status code instead of just 100? Yes, that should be possible.